File: C prog/day4/array1.c
#include <stdio.h>
int main()
{
    int value, n;
    int arr[n];
    printf("Enter the size of array:\n");
    scanf("%d", &n);
    printf("Enter the %d element \n", n);

    for (int i = 0; i < n; i++)
    {
        scanf("%d", &value);

        arr[i] = value;
    }

    printf("The Elements are \n");
    for (int j = 0; j < n; j++)
    {
        printf("%d \n", arr[j]);
    }
}

File: eclipse basic java/Oppsinjava/src/Array2d/Array1.java
package Array2d;

import java.util.Scanner;

public class Array1 {
	public static void main(String[] args) {
		
		int [][] arr = new int[3][4];
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter elements");
		for(int i=0; i<3 ;i++) {
			for(int j=0; j<4; j++) {
				arr[i][j] = sc.nextInt();
			}
			
		}
		
		
		System.out.println("The array elements are");
		for(int i=0; i<3 ;i++) {
			for(int j=0; j<4; j++) {
				
				System.out.print(arr[i][j]+ " ");
				}
			System.out.println();
			}
		
		int sum =0;
		for(int i=0; i<3 ;i++) {
			for(int j=0; j<4; j++) {
				
				sum= sum+arr[i][j];
				
				}
			
			}
		System.out.print( " Sum of array element is " +sum);
		
		
		
			
		
	}

}
